apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -956,286 +1203,374 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
-                                                            />
-                                                        </Form.Item>
-                                                    </Col>
-                                                )}
-                                                {
-                                                    !isValidBeforeAfterSendAt && (
-                                                        <Alert
-                                                            type='error'
-                                                            showIcon
-                                                            message={ValidBeforeErrorMessage}
-                                                            description=' '
-                                                        />
-                                                    )
-                                                }
                                                 {
-                                                    newsItemCountAtSameDay >= 5 && (
-                                                        <Alert
-                                                            type='warning'
-                                                            showIcon
-                                                            message={ToManyMessagesMessage}
-                                                            description=' '
-                                                        />
+                                                    !!formInfoColSpan && newsItemScopesNoInstance.length > 0 && (
+                                                        <Col span={formInfoColSpan}>
+                                                            <RecipientCounter newsItemScopes={newsItemScopesNoInstance}/>
+                                                        </Col>
                                                     )
                                                 }
                                             </Row>
                                         </Col>
-                                    </Row>
-                                </Col>
-                                {isFunction(ActionBar) && (
-                                    <Col span={24} style={MARGIN_TOP_44_STYLE}>
-                                        <ActionBar
-                                            handleSave={handleSave}
-                                            isLoading={isLoading}
-                                            form={form}
+                                    </>
+                                ) }
+
+                                {
+                                    ( getStepTypeByStep(currentStep) === 'sharingApp' ) && (
+                                        <NewsItemSharingForm
+                                            onIsValidChange = { (isValid) => { console.log(`Form is valid: ${isValid}`) } }
```

Comment:
This was part of functionality that I wanted to refactor :-( 

Thing is, right now steps that handle Next and Skip buttons are inside NewsSharingForm component. This is a bit strange, because steps are handled in BaseNewsForm component. I left a TODO and removed console.log